src/chunks.rs

```diff
@@ -643,6 +643,18 @@ impl<T> ReadChunk<'_, T> {
         )
     }
 
+    /// Returns two mutable slices for reading from the requested slots.
+    ///
+    /// This has the same semantics as [`as_slices()`](ReadChunk::as_slices), except that
+    /// returns mutable slices and requires a mutable reference to the chunk.
```

Comment:
I have the feeling that users might be confused as to why there are now two separate functions `as_slices()` and `as_mut_slices()`. It might also not be obvious why it makes sense to have a writable slice for a "reading" operation.

Can you please add some documentation explaining that `as_slices()` is the normal thing to use and that `as_mut_slices()` is only useful for niche use cases, as you explained in #99?

Reply:
Good point, updated the docs to clarify